### ISSUE TITLE
Implement an openra-mod:// URI for installing third party mods from a website.

### ISSUE THREAD

**Issue by pchote** [OpenRA/OpenRA#9029]
Like #9027, but for mods.  Also depends on #8992 and #8991.  This would make it possible for third party mods to include an "Install for OpenRA" button on their websites.

The utility command should accept a url to the mod version/metadata file proposed in #9028 and then show a dialog with basic information (mod title / version / author) and a warning if there no compatible versions for the current version of OpenRA or if the latest version available is earlier than the indicated `LatestRelease`.  The command should download the mod to a temporary location, and then inspect its mod.yaml.  If it defines / includes any custom dlls it should show a prominent warning and then ask the player if they want to continue installing the mod.
##

**Comment by Mailaender** [comment]
Implemented in https://github.com/OpenRA/OpenRA/pull/11870.

**Comment by pchote** [comment]
This is about web-browser support, not running locally.

**Comment by pchote** [comment]
Superseded by the more realistic #12554, with mods shipping their own OS-native installers generated from our mod template scripts.
